Editor/Data/Type/DefaultObservableListDrawer.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Zinnia.Data.Type
+{
+    using UnityEngine;
+    using UnityEditor;
+    using System;
+    using Zinnia.Utility;
+    using Zinnia.Data.Collection.List;
+
+    [CustomPropertyDrawer(typeof(ObservableList), true)]
+    public class DefaultObservableListDrawer : PropertyDrawer
+    {
+        protected const string collectionLabel = "{0} Elements";
```

Comment:
```suggestion
        protected const string collectionLabel = "Elements";
```
Thanks to the proper nested drawing of the foldout it feels like we can get rid of the duplication in the naming.